app/Http/Controllers/Consumables/ConsumableCheckoutController.php

```diff
@@ -48,6 +49,14 @@ public function store(Request $request, $consumableId)
             return redirect()->route('consumables.index')->with('error', trans('admin/consumables/message.not_found'));
         }
 
+        $request->validate([
+            "totalnum" => "required|regex:/^[0-9]*$/|max:{{$consumable->numRemaining()}}"
```

Comment:
Blade syntax is only for blades - I don’t think this will work here

Reply:
(Even if it did, we’d want a custom validator here, so we could localize the strings)

Reply:
any advice you might have for the proper validation syntax location?

Reply:
string will be updated to support localization

Reply:
for now,  strings on validator has been moved to admin/consumables/message.
so far checking for these 3 validations work. but whenever I switch my locale to different language,  it doesn't switch. strange..